FAQ.md

```diff
@@ -0,0 +1,44 @@
+# Frequently asked questions
+
+## Getting started
+
+First you need to clone the repository. You can do this with
+
+```bash
+git clone https://github.com/WhatsApp/proxy.git
+```
+
+## Common issues
+
+### (1) The container won't build on Windows with `set_public_ip_and_start.sh: Not found`
+
+This is likely a line encoding issue since the application is expecting unix-style line
+encoding (EOL not CRLF). This resolved in PR [72](https://github.com/WhatsApp/proxy/pull/72)
+and you should just need to pull the latest changes and try again.
+
+### (2) I want to create share my proxy to the community
+
+This is great! We have create a dedicated GitHub issue to share these proxies. Access it here: [Issue #92](https://github.com/WhatsApp/proxy/issues/92)
+
+### (3) My proxy isn't accessible publicly
+
+Some common problems to investigate
+
+1. Are the necessary ports open on your host?
+2. If running in a cloud, are the necessary ports open on the cloud provider's firewall.
+3. Can you access the statistics port (8199) locally? (at `http://127.0.0.1:8199`) If not, can you check the health of the container (if it's up and running) with `docker ps`?
+
+### (4) What is the port configuration for this service + the client?
+
+The client will attempt to connect by default to port 443 is only a host (ip or domain name) is specified in the app. If you specify a port with `:PORT` then that will override the default.
+
+You can also re-map the ports exposed from the proxy to whatever you want. For example if you have a different service running on port 80, you can send the proxy container's port 80 to port 8081 for example. You can do this by changing the port mapping from `80:80` to `8081:80`. 
```

Comment:
this is probably right but just a note to double-check that it isn't inverted:

to send port 80 to 8081 seems like it might require `80:8081`

either way, a link directly into source of where this change would be required ([such as here](https://github.com/WhatsApp/proxy/blob/de2477123aa66dd337a00baf51b6878fd9da128b/proxy/ops/docker-compose.yml#L14)) might be helpful for some.

Reply:
ah in this case it's {host_port}:{container_port}. We aren't changing the container ports, but what gets actually exposed to the world should be the first number.